Adjusted note on tags + typos (#65)
mulesoft/docs-visualizer: modules/ROOT/pages/use-tags-in-visualizer.adoc

== Setting the Tag Through Properties

> You can automate adding tags to a node by using `anypoint.platform.visualizer.tags=<name>, <name>`, either in the application's properties file or at deployment time through the Runtime Manager UI or API.

Reviewer: Can a node have more than one tag? If not, then "adding tags to nodes".

Author: ```suggestion
You can automate adding tags to nodes by using `anypoint.platform.visualizer.tags=<name>, <name>`, either in the application's properties file or at deployment time through the Runtime Manager UI or API. Note that this is an application property, not a server property.
```

[NOTE]

> Setting the tag using the property file is not currently supported for Mule 4 runtimes. You can only use the Runtime Manager UI for Cloudhub and Runtime Fabric deployments. 

Reviewer 2: this will be fixed with AVE-319 . do we need to document this ?

Author: Until it is fixed, I'd say yes.

> Setting the tag using the property file is not currently supported for Mule 4 runtimes. You can only use the Runtime Manager UI for Cloudhub and Runtime Fabric deployments. 

Reviewer: Setting which tag? Perhaps "setting a tag"?

Author: ```suggestion
Setting a tag using the property file is not currently supported for Mule 4 runtimes. You can only use the Runtime Manager UI for Cloudhub and Runtime Fabric deployments. 
```

> Note that this is an application property, not a server property.

Reviewer 2: this line "Note that this is an application property, not server property" . Should we move it out of the note into the main section ( that's how it is in layers.adoc )

Author: We can, yes.

> Note that this is an application property, not a server property.

Author: ```suggestion
```

image::properties-tags.png[]

[[set-tag-ui]]

> == Setting the Tag Through the UI

Reviewer: Which tag? Perhaps "Setting a Tag Through the UI"?
Also, for translators, would "by using" be more comprehensible than "through"?

Author: ```suggestion
== Set Tags Using the UI
```
